galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.manager/src/main/java/dev/galasa/cicsts/ITsqFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.cicsts;
+
+import javax.validation.constraints.NotNull;
+public interface ITsqFactory {
```

Comment:
I prefer ITSQFactory or TSQFactory. 'I' Prefix is optional, but has been used pretty consistently in Galasa, but I'm personally not wedded to it. Given that the implementation is internal to the bundle and non-reachable, I think that should be TSQFactoryImpl or similar ?

Reply:
Hi @techcobweb , I can find that all the interfaces have name starting with 'I' and  the implementation of ITsqFactory.java is named as TsqFactoryImpl.java.  I thought this is the naming convention followed in Galasa.  Please advise if I need to change this or keep it similar to the existing file names in Galasa?